### DISCUSSION TITLE
Inline text highlighting in TOC section.

### DISCUSSION THREAD

**Discussion by UltraInstinct05** [squidfunk/mkdocs-material#3831] [Q&A] [answered]
Is it possible to render inline code highlighting (The one you get with ``, 2 bacticks) in the Table of Contents section ?

Thanks !

**Comment by squidfunk** [comment, marked as answer]
Sadly, the Python Markdown `toc` extension doesn't allow that. However, I prototyped a fork of the extension some time ago which makes this possible, but I haven't had the time to finish it. Thus, it might be possible soon, but currently no ETA.

**Reply by UltraInstinct05** [reply]
No problem at all. It would be a nice addition to have nonetheless. Thanks for your great work on this awesome tool. Writing technical docs/guidebooks/handbook sort of stuff with `mkdocs-material` is a joy !
